.github/workflows/builders.yaml

```
@@ -1,32 +1,11 @@
-name: presubmit-pr-check
-description: Presubmit Pull Request Verification
+name: builders
+description: Presubmit build and deploy from sources tests
```

actionlint (1.7.8)

unexpected key "description" for "workflow" section. expected one of "concurrency", "defaults", "env", "jobs", "name", "on", "permissions", "run-name"

(syntax-check)